manual/Data_creation_and_modification/Adding_documents_to_a_table/Adding_documents_to_a_real-time_table.md

```diff
@@ -269,6 +269,8 @@ If you attempt to INSERT multiple rows with different, incompatible value types
 * uint -> bigint -> float (this may cause some precision loss)
 * string -> text
 
+The auto schema mechanism does not support creating [KNN](../../Searching/KNN.md#Configuring-a-table-for-KNN-search) tables, so you cannot insert `float_vector` data into an automatically created table. If you still want to store `float_vector` values in a regular table, you can insert them using the same syntax as for JSON data.
```

Comment:
Sounds a little vague:
- what is "KNN tables"?
- "you cannot insert `float_vector` data into an automatically created table" - it's better to say you can't create a table with fields of type `float_vector`
- "If you still want to store `float_vector` values in a regular table". What's meant by "float_vector" values?
- "you can insert them using the same syntax as for JSON data" - what's the syntax for JSON data?